README.md

```diff
@@ -25,7 +25,8 @@
 - Twitter: [@isaacpitwa](https://twitter.com/isaacpitwa)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/isaac-pitwa)
 
-
+## Attribution
+Original design Idea by  [Cindy Shin](https://www.behance.net/adagio07)
 ## 🤝 Contributing
 
```

Comment:
[OPTIONAL] It would be better if you add acknowledgments to your README. If you have used anyone else's work (designs, images, etc) that has a copyright that requires acknowledgment, you might want to add that to the acknowledgment section. You can also acknowledge any other developers or institutions who helped with the project. Here 👇 is an example of a really good "Achnowledgments" section:

![https://user-images.githubusercontent.com/77190203/152332184-14fe5aa2-11c0-400d-842f-fb07c1f52d81.png](https://user-images.githubusercontent.com/77190203/152332184-14fe5aa2-11c0-400d-842f-fb07c1f52d81.png)